ui/src/main/webapp/app/query/view-builder/view-builder.html

```diff
@@ -8,42 +8,28 @@ <h3 class="md-title section-title">Add Filters</h3>
     <div ng-show="ctrl.showBuilder">
         <div layout="row" layout-padding>
             <md-input-container flex>
-                <label>Only include these entities</label>
-                <md-select id="view-entities" class="no-ellipsis" multiple
-                           md-selected-text="ctrl.createViewElementsLabel(ctrl.model.viewEntities, 'entities')"
-                           ng-model="ctrl.model.viewEntities"
-                           ng-change="ctrl.onElementGroupChange('entity')"
-                           data-md-container-class="searchable-select">
+                <label>Only include these entity types</label>
+                <md-select id="view-entities" class="no-ellipsis" multiple md-selected-text="ctrl.createViewElementsLabel(ctrl.model.viewEntities, 'entity types')"
+                    ng-model="ctrl.model.viewEntities" ng-change="ctrl.onElementGroupChange('entity')"
+                    data-md-container-class="searchable-select">
                     <md-select-header class="search-header">
-                        <input
-                                ng-model="ctrl.entitySearchTerm"
-                                type="search"
-                                class="search-box md-text"
-                                placeholder="Search for an entity group...">
+                        <input ng-model="ctrl.entitySearchTerm" type="search" class="search-box md-text" placeholder="Search for an entity group...">
                     </md-select-header>
-                    <md-option ng-value="entity"
-                               ng-repeat="(entity, detail) in ctrl.schemaEntities | schemaGroupFilter:ctrl.entitySearchTerm">
+                    <md-option ng-value="entity" ng-repeat="(entity, detail) in ctrl.schemaEntities | schemaGroupFilter:ctrl.entitySearchTerm">
                         <div class="md-subhead">{{entity}}</div>
                         <div class="md-body-1">{{detail.description}}</div>
                     </md-option>
                 </md-select>
             </md-input-container>
             <md-input-container flex>
                 <label>Only include these edges</label>
```

Comment:
Could this one be as well?

Reply:
Made that change and it doesn't seem to affect anything negatively that I can tell, so I'll add it to the changes. Just did the necessary changes to get the result beforehand, but thanks for pointing this out.